bindings/src/main/java/org/commonjava/rwx/binding/internal/reflect/ReflectionUnbinder.java

```diff
@@ -445,7 +445,6 @@ else if ( binding.getFieldType().isArray() || Collection.class.isAssignableFrom(
                     }
 
                     logger.debug( "Attempting to coerce raw value of: {} using: {}", value, coercion );
-                    value = coercion.toString( value );
```

Comment:
Shouldn't also the log message above go away along with the action? I'm not saying it is ok to drop the toString call, just noticed that it would be very confusing to find a message in the log saying it did something while the action itself is not performed.